web/client/plugins/longitudinalProfile/enhancers/useFile.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+import { compose, mapPropsStream, withHandlers } from 'recompose';
+
+import {reprojectGeoJson} from "../../../utils/CoordinatesUtils";
+import {selectLineFeature} from "../../../utils/LongitudinalProfileUtils";
+
+/**
+ * Enhancer for processing json file with features
+ * Flatten features and then selects first line feature to build longitudinal profile out of it
+ */
+export default compose(
```

Comment:
Was recompose really needed in this new file?

Reply:
it was inherited from existing codebase